docs/admin/index.md

```diff
@@ -58,6 +58,14 @@ explicitly-managed secret.
 - [GitHub](https://github.com/): w3c
   - [@plehegar](https://github.com/plehegar)
   - [@sideshowbarker](https://github.com/sideshowbarker)
+- [Azure Pipelines](https://dev.azure.com/web-platform-tests/wpt)
```

Comment:
Yep, there's a mess of different default teams and access levels, and leftovers from trying different things.

Would you like me to change something to get this PR merged?

Reply:
I'd like some sort of explanation as to what's going on. Who has access to what—do we care that some users are stakeholders and some are basic in the org? Should we simplify the groups we have within the project, so it's more obvious who has access to what?

Reply:
I don't want to make any changes as part of documenting this, because to be confident in any change requires understanding the mess. I can document that it's a bit messy and that nobody understands it, however.